.gitpod.dockerfile

```diff
@@ -5,22 +5,22 @@ ENV TRIGGER_REBUILD 3
 
 # Install custom tools, runtime, etc.
 RUN sudo apt-get update \
-    # window manager
-    && sudo apt-get install -y jwm \
-    # electron
-    && sudo apt-get install -y libgtk-3-0 libnss3 libasound2 libgbm1 \
-    # native-keymap
-    && sudo apt-get install -y libx11-dev libxkbfile-dev \
-    # keytar
-    && sudo apt-get install -y libsecret-1-dev \
-    && sudo rm -rf /var/lib/apt/lists/*
+  # window manager
+  && sudo apt-get install -y jwm \
+  # electron
+  && sudo apt-get install -y libgtk-3-0 libnss3 libasound2 libgbm1 \
+  # native-keymap
+  && sudo apt-get install -y libx11-dev libxkbfile-dev \
+  # keytar
+  && sudo apt-get install -y libsecret-1-dev \
+  && sudo rm -rf /var/lib/apt/lists/*
```

Comment:
It looks like your IDE autoformatted this code. Would you mind reverting the formatting changes? We usually use a tab size of 4 spaces everywhere.